src/main/java/siri/Duke.java

```diff
@@ -0,0 +1,182 @@
+package siri;
+
+import siri.exception.AddTaskIndexOutOfBounds;
+import siri.exception.MarkerArrayIndexOutOfBoundsException;
+import siri.exception.UnknownCommandException;
+import siri.task.Deadline;
+import siri.task.Event;
+import siri.task.Task;
+import siri.task.ToDo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private static Task[] tasks = new Task[100];
+    private static int indexOfTask = 0;
+
+    public static void drawLine() {
+        System.out.println("=====================================================================================");
+    }
+
+    public static void createSiriChatBox() {
+        System.out.println("Siri:\n");
+    }
+
+    public static void createUserChatBox() {
+        System.out.println( "user: \n");
+    }
+
+
+    public static void greet() {
+        //greet and ask name
+        drawLine();
+        createSiriChatBox();
+        System.out.println("Hey, I'm Siri\n" + "What can I do for you?");
+        drawLine();
+    }
+
+    public static void sayGoodbye() {
+        //say goodbye with name
+        drawLine();
+        createSiriChatBox();
+        System.out.println(">o< Goodbye! Hope to see you again soon! >o<");
+        drawLine();
+    }
```

Comment:
These comments are not neccessary and should be removed if it is just a note to yourself. Instead, you can make a description for the methods using Javadoc